projects/testing-library/tests/render.spec.ts

```diff
@@ -533,3 +534,46 @@ describe('configureTestBed', () => {
     expect(configureTestBedFn).toHaveBeenCalledTimes(1);
   });
 });
+
+describe('inputs and signals', () => {
+  @Component({
+    selector: 'atl-fixture',
+    template: `<span>{{ myName() }}</span> <span>{{ myJob() }}</span>`,
+  })
+  class InputComponent {
+    myName = input('foo');
+
+    myJob = input('bar', { alias: 'job' });
+  }
+
+  it('should set the input component', async () => {
+    await render(InputComponent, {
+      inputs: {
+        myName: 'Bob',
+        job: aliasedInputWithValue('Builder'),
+      },
+    });
+
+    expect(screen.getByText('Bob')).toBeInTheDocument();
+    expect(screen.getByText('Builder')).toBeInTheDocument();
+  });
+
+  it('should typecheck correctly', async () => {
+    // @ts-expect-error - myName is a string
+    await render(InputComponent, {
+      inputs: {
+        myName: 123,
```

Comment:
It would be great if TypeScript showed the error on this line rather that on the `render` call.

I think there was a way to trick it into showing it here, but I forgot how.

Reply:
I was also looking at it, but without success. I think it's fine to keep it as is.

Reply:
Someone on the Typescript discord might be able to improve this further: https://discord.gg/typescript but I didn't have time to make it into a simple repro.

I vaguely remember there were some tricks to move the error downwards.